tests/cli/test_work_queues.py

```diff
@@ -327,10 +327,31 @@ def test_pause_bad_pool_name(
             expected_code=1,
         )
 
+    def test_pause_without_specifying_pool_name_without_confirmation(
+        self,
+        work_queue,
+    ):
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name}",
+            expected_code=1,
+        )
+
+    def test_pause_without_specifying_pool_name_with_confirmation(
+        self,
+        work_queue,
+    ):
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name}",
+            user_input="Y",
+            expected_code=0,
+        )
```

Comment:
We could add a test that "N" input aborts as expected and has the right message.

Reply:
Great input! I've added a unit test to address your concern in latest commit

Reply:
Awesome! One last thing is asserting `expected_output_contains` when selecting "Y" but otherwise looks good!